Error instantiating goakit logger in example server

I'm using goa [grpc_go_1_26](https://github.com/goadesign/goa/tree/grpc_go_1_26) branch and goakit plugin from [v3.2.6](https://github.com/goadesign/plugins/tree/v3.2.6).

When I try to build generated examples, I get an error:
```console
> go build -o ./bin/example.exe .\cmd\example\
cmd\example\grpc.go:30:33: cannot use logger (type "github.com/go-kit/kit/log".Logger) as type *"log".Logger in argument to "goa.design/goa/v3/middleware".NewLogger
```

Cause of this error is that goakit generator in `gokitifyExampleServer()` don't replace `middleware.NewLogger(logger)` to `logger`. As it is done for example in zaplogger's `updateExampleFile()`.

My API design definitions:
```go
package design

import (
	. "goa.design/goa/v3/dsl"
	_ "goa.design/plugins/v3/goakit"
)

var (
	_ = API("example", func() {
		Title("example")
		Version("1.0")
		Description("example api description")
		Server("example", func() {
			Description("example server description")
			Services("svc")

			Host("development", func() {
				Description("Development host")
				URI("grpc://localhost:50051")
			})
		})
	})

	_ = Service("svc", func() {
		Description("svc desc")

		Method("mtd", func() {
			Payload(MtdReq)

			Result(MtdResp)
			Error("NotFound")
			Error("BadRequest")

			GRPC(func() {
				Response("NotFound", CodeNotFound)
				Response("BadRequest", CodeInvalidArgument)
			})
		})
	})

	MtdReq = Type("MtdRequest", func() {
		Description("MtdRequest desc")
		Attribute("att", ArrayOf(String), "att desc", func() {
			Example([]string{
				"R",
				"T",
			})
			Meta("rpc:tag", "1")
		})
	})

	MtdResp = Type("MtdResponse", func() {
		Description("MtdResponse desc")
		Attribute("att", MapOf(String, Boolean), "att desc", func() {
			Example(map[string]bool{
				"R": true,
				"T": true,
			})
			Meta("rpc:tag", "1")
		})
	})
)

```

--- a/goakit/generate.go
+++ b/goakit/generate.go
@@ -88,6 +88,7 @@ func gokitifyExampleServer(genpkg string, file *codegen.File) {
 			hasLogger = strings.Contains(s.Source, "*log.Logger")
 		}
 		s.Source = strings.Replace(s.Source, "*log.Logger", "log.Logger", -1)
+		s.Source = strings.Replace(s.Source, "adapter = middleware.NewLogger(logger)", "adapter = logger", 1)
 		codegen.AddImport(file.SectionTemplates[0], &codegen.ImportSpec{Path: "fmt"})
 		s.Source = goaLoggerRegexp.ReplaceAllString(s.Source, "logger.Log(\"info\", fmt.Sprintf(${1}))")
 

